Header  header

int16   gnss_week
float64 gnss_time

float64 gyroscope_x
float64 gyroscope_y
float64 gyroscope_z

float64 acceleration_x
float64 acceleration_y
float64 acceleration_z

float64 temperature